### ISSUE TITLE
Display name of current project

### ISSUE THREAD

**Issue by windsource** [hetznercloud/cli#362]
It would be nice if `hcloud` could display the active project name in Hetzner cloud.

I am working with several projects in Hetzner cloud and have created several contexts for that. With `hcloud context active` I am able to show the current context. But that does not work anymore when the environment variable `HCLOUD_TOKEN` ist set which I use for scripting. 

I am not sure if the current Hetzner Cloud API supports that feature but it would be nice to have it in order not to deploy to a wrong project. Finally I want to integrate the display of the project name in the shell prompt ([Spaceship ZSH](https://github.com/spaceship-prompt/spaceship-prompt)).

**Comment by c33s** [comment]
this is my workaround on windows but maybe it helps you:

TLDR:  @windsource try [direnv][3]

/cc @LKaemmerling any chance you rethink your position on not integrating the config file? the windows support of direnv still isn't that good (issues: [windows support][5], [powershell support][4])
having a simple config file would fit into the KISS principle and would allow more flexibility and usage-diversity (the one-thing-fits-all mentality in the IT gets worse from year to year).

---

some time ago i requested the feature that the [context set by a config file][1], this got implemented with the pullrequest [Load context from .hcloud-context file][2]. sadly the PR was never merged in the main branch. instead it was suggested to use [direnv][3]. for me direnv doesn't work (no smooth integration, more dependencies to maintain, no good windows support). i stick with the version of hcloud which was compiled with this feature but maybe direnv works for you.

![image](https://user-images.githubusercontent.com/649209/145736575-b9a44eae-651f-4a3a-880a-7fe6938cfcd8.png)


for the version which supports the config file, i implemented a [small lua script for cmder][6]. the following code is an adapted version, it shows the git branch, the terraform and the hcloud context:
```lua
local color = require("color")

local function is_terraform()
    for dir in io.popen([[dir ".\" /b /ad]]):lines() do
        -- print(dir)
        if dir == ".terraform" then
            return true
        end
    end

    return false
end

local function hcloud_prompt_filter()
    if is_terraform() then
        f = assert (io.popen ("hcloud context active"))
        for line in f:lines() do
            hcloud_context = '(hcc: ' .. line .. ')'
            break
        end -- for loop
        f:close()
        
        f = assert (io.popen ("hcloud context active"))
        for line in f:lines() do
            if string.match(line, "production") then
                prompt_color = color.CYAN
            elseif string.match(line, "development") then
                prompt_color = color.GREEN
            else
                prompt_color = color.RED
            end
            break
        end -- for loop
        
        hcloud_context = color.color_text(hcloud_context, prompt_color)
        clink.prompt.value = clink.prompt.value:gsub("\n", hcloud_context .. "\n")

        return false
    end
end

clink.prompt.register_filter(hcloud_prompt_filter, 1)
```
[1]: https://github.com/hetznercloud/cli/issues/154
[2]: https://github.com/hetznercloud/cli/pull/162
[3]: https://direnv.net/
[4]: https://github.com/direnv/direnv/issues/124
[5]: https://github.com/direnv/direnv/issues/343
[6]: https://dev.to/c33s/how-to-add-a-custom-prompt-to-cmder-gb2
